.github/velocity.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+from datetime import date, datetime
+import csv
+import os
+
+import pandas as pd
+
+from storypoints import ZenHubAPI, GitHubAPI
+
+DEVTEAM_PROJECTS = [
+    'cdh-web', 'winthrop-django', 'django-pucas', 'djiffy', 'django-annotator-store',
+    'derrida-django', 'mep-django', 'viapy',
+    'ppa-django', 'parasolr', 'pemm-scripts', 'startwords'
+]
+
+def get_closed_issues():
+    # Get APIs
+    ghub = GitHubAPI(os.environ['GITHUB_USERNAME'], os.environ['GITHUB_API_TOKEN'])
+    zhub = ZenHubAPI(os.environ['ZENHUB_API_TOKEN'])
+
+    # Read existing CSV
+    filename = 'data/issues.csv'
+    if os.path.exists(filename):
+        df = pd.read_csv(filename, parse_dates=['closed'])
+        # find the latest date already read
+        date_since = df['closed'].max()
```

Comment:
`issues.csv` is updated by finding the latest date in the CSV and searching the GitHub API since that date. The new issues are appended to the CSV